x/distribution/types/common_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// nolint:deadcode,varcheck
+//nolint:deadcode,varcheck
```

Comment:
if there is deadcode, cannot we simply remove it?

Reply:
let's try